### PR TITLE
Add benchmark CI

### PR DESCRIPTION
**Pull request by KotaTakahashi9320**
# Summary
Add benchmark CI
# Purpose
To keep up to date with the latest QuickMPC run times
# Contents
Benchmark runs when release is done
# Testing Methods Performed
Execute in local

<details>
<summary>result log</summary>
Run with small data for debug
スクリーンショット 2023-06-09 17 10 15

### CODE REVIEW

File: benchmark_src/test/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      dockerfile: ./Dockerfile
+      target: medium
+    volumes:
+      - target: /libclient/src
+        source: ./src
+        type: bind
+      # TODO sourceをdataに変更(gcsをmountしたディレクトリがdockerにmountできなかったためtest_dataを作成している)
+      - target: /libclient/data
+        source: ./test_data
+        type: bind
```
[the remark this answers is not in the document]
Comment (KotaTakahashi9320) [reply benchmark_src/test/docker-compose.yml:19]: Error when mounting, to be fixed in the future.

**Comment by KotaTakahashi9320** [comment]
@come2ry 
Corrected based on comments received.
I would appreciate your confirmation.

**Review by come2ry — APPROVED** [review]
LGTM

**Comment by KotaTakahashi9320** [comment]
Results in 10 million cases
スクリーンショット 2023-06-14 13 08 22

**Pull request merged**